catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/SearchCommand.java

```diff
@@ -117,11 +117,12 @@ private Object executeSearchStore(Filter filter) throws Exception {
     long end = System.currentTimeMillis();
 
     console.println();
+    final long hits = catalogProvider.query(new QueryRequestImpl(query)).getHits();
```

Comment:
I like that this  is simple and doesn't add much complexity.. so not sure if its worth it to do the following: we might consider setting the page size on this query to like 1 item to reduce network overhead for this query since it's purely going out for the hits.

Though -- i'm fine with leaving this as is and addressing it if /when it ever does become an issue